doc/manual/citing_aspect.bib

```diff
@@ -1941,4 +1941,32 @@ @article{se-14-1155-2023
 PAGES = {1155--1168},
 URL = {https://se.copernicus.org/articles/14/1155/2023/},
 DOI = {10.5194/se-14-1155-2023}
-}
\ No newline at end of file
+}
+
+@Article{Dannberg2023,
+author={Dannberg, Juliane
+and Chotalia, Kiran
+and Gassm{\"o}ller, Rene},
+title={How lowermost mantle viscosity controls the chemical structure of Earth's deep interior},
+journal={Communications Earth {\&} Environment},
+year={2023},
+month={Dec},
```

Comment:
Bibtex is weird: If you provide a month *name*, you can't use curly braces around it. If you provide a month *number*, you can use the braces, however.